src/scripts/main.js

```diff
@@ -1,7 +1,94 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const container = document.querySelector('.container');
+const gameField = document.querySelector('.game-field');
+const button = container.querySelector('.button');
+const gameScore = container.querySelector('.game-score');
+
+const fieldRows = [...gameField.querySelectorAll('.field-row')];
+const fieldCells = fieldRows.map((row) => [...row.children]);
+
+const messages = {
+  idle: container.querySelector('.message-start'),
+  lose: container.querySelector('.message-lose'),
+  win: container.querySelector('.message-win'),
+};
+
+function fillGameField(state) {
+  state.forEach((row, i) => {
+    row.forEach((cell, j) => {
+      const currentCell = fieldCells[i][j];
+
+      currentCell.className = !cell
+        ? 'field-cell'
+        : `field-cell field-cell--${cell}`;
+      currentCell.innerHTML = !cell ? '' : cell;
+    });
+  });
+}
+
+function showMessage() {
+  const gameStatus = game.getStatus();
+
+  for (const key in messages) {
+    if (Object.hasOwnProperty.call(messages, key)) {
+      const message = messages[key];
+
+      if (message) {
+        message.classList.toggle('hidden', key !== gameStatus);
+      }
+    }
```

Comment:
Consider using `Object.entries` for iterating over the `messages` object. It provides a cleaner syntax and directly gives you access to the key and value without having to use `Object.hasOwnProperty.call`.